src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -223,6 +223,8 @@ frmelmnts.lbl.anncmnttblseen = Seen
 frmelmnts.lbl.anncmnttblsent = Sent
 frmelmnts.lbl.attributions = Attributions
 frmelmnts.lbl.author = Author
+frmelmnts.lbl.authorofsourcecontent = Author of Source Content
+frmelmnts.lbl.contentderivedfrom = This Content is Derived From
```

Comment:
frmelmnts.lbl.contentcopiedtitle = This content is derived from 
this is already available so why do we need one more

Reply:
Removed.